api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export function authMiddleware(req, res, next) {
+  /* your code here */
+  const { authorization } = req.headers;
+  const token = authorization.split('Bearer ')[1];
+  if (!authorization || !token) {
```

Comment:
Here you are checking that `authorization` is defined, but it's too late! You already crashed up in line 8.